### PR TITLE
Add NK-21.

### PR DESCRIPTION
**Pull request by ahmedcharles**

### CODE REVIEW

**Comment by SirKeplan** [comment]
NK-21 was not restartable in space. (probably not relightable on ground tests either, if it was at all like the NK-15)

**Comment by ahmedcharles** [comment]
@SirKeplan Changed.

File: GameData/RealismOverhaul/Engine_Configs/NK9V.cfg [hunk 1]
```diff
@@ -66,6 +66,44 @@
 		}
 		CONFIG
 		{
+			name = NK-21
+			description = NK-9V rerated for N1 Block V use. Relighable. No gimbal, differential throttle.
```
Comment (SirKeplan) [inline GameData/RealismOverhaul/Engine_Configs/NK9V.cfg]: still says "relightable"

File: GameData/RealismOverhaul/Engine_Configs/NK9V.cfg [hunk 2]
```diff
@@ -197,6 +235,19 @@
 		techTransfer = NK-9:50
 	}
 }
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[NK-21]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+{
+	TESTFLIGHT
+	{
+		name = NK-21
+		ratedBurnTime = 370 // http://www.astronautix.com/n/n1blockv.html
```
Comment (SirKeplan) [inline GameData/RealismOverhaul/Engine_Configs/NK9V.cfg]: The main difference between the NK-19 and NK-21 seems to be with the gimbal/thrust mount. so rated burn time should be the same as the NK-19 here.

**Comment by ahmedcharles** [comment]
Updated with the requested changes.

**Pull request merged**